### DISCUSSION TITLE
Kney'tehab

### DISCUSSION THREAD

**Discussion by milz19** [L-Sherry/French-CC#13] [General]
oubli de traduction sur le pnj de la quete "haine de caisse" le pnj "Kney' tehab" de Ba'kii kum parle un moment en anglais quand il te donne la quete. merci

**Comment by L-Sherry** [comment]
Bonjour,

Vous utilisez quelle version de CrossCode ? La traduction est prévue pour 1.4.2-2, et une partie du texte de cette quête a changé dans cette version :

https://github.com/L-Sherry/French-CC/commit/a381d5680023928a8e7e3883ae0ddc95ab817c28#diff-276ff8b0782aa4bde96b4ec04d7d4a5ad453652dcf7eab7569be7253f3c9a379

**Reply by milz19** [reply]
Bonjours,
j'utilise la même version. 1.4.2-2 avec dlc new home et ninja skin. Je n'ai pas trouvé sa personnellement dérangeant c'était juste pour aider. j'ai même parfois vu les quêtes de la mairie qui n'était pas traduit et un petit bout du dlc, je dois tout signaler? parce que si oui je peu toujours continuer mon ng++ xD

**Reply by L-Sherry** [reply]
S'il y a des morceaux qui se retrouvent en anglais c'est que la traduction ne correspond pas à la version du jeu.

Normalement à ce state absolument tout est traduit dans 1.4.2-2 et la correspondance entre french-cc 1.3.0 et crosscode 1.4.2-2 a été automatiquement vérifiée. Je n'ai pas connaissance qu'il y ai plusieurs versions 1.4.2-2 dans la nature. Et à ce que je vois, les autres moddeurs qui traduisent CrossCode dans d'autres langues non plus.

Par curiosité, votre version de CrossCode vient d'où ? Steam ? GOG ? Itchio ?

**Comment by milz19** [comment]
steam

**Comment by milz19** [comment]
...bon j'ai regardé car tu m'avais mis le doute et étrangement dans mon
dossier téléchargement du 5septembre que j'ai pris sur le mod steam j'avais
la version 1.2, je viens de prendre la 1.3 xD j'ai rien compris
d'ou il sort se 1.2
